### PR TITLE
Add Liya.app v3.0.5

### PR DESCRIPTION
**Pull request by arielallon**
Adding a cask for Liya
http://cutedgesystems.com/software/liya/
Liya is a GUI for SQLite (as well as Postgres and Mysql)

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/liya.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'liya' do
+  version '3.0.5'
+  sha256 '9441df5c2790e26b3496d7dde666c52dd8c6a432799fae0221c52490ca079997'
+
+  url 'https://cutedgesystems.com/downloads/Liya.zip'
+  name 'Liya'
+  homepage 'http://cutedgesystems.com/software/liya/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/liya.rb]: Please either set a license or add [a comment](https://github.com/caskroom/homebrew-cask/blob/6a66da2e01e5d9b7b71921045aaeb810e22ff206/Casks/cheetah3d.rb#L7).

**Comment by arielallon** [comment]
Hey @vitorgalvao thanks for the super-fast and constructive feedback!
Made some updates, let me know what you think.

**Comment by fanquake** [comment]
@arielallon Can you squash your commits?

**Comment by ndr-qef** [comment]
I took the liberty of cleaning up the branch and merging as e0c1847. Thanks.

**Comment by arielallon** [comment]
Thanks

**Pull request closed without merging**